examples/path_mappings/BUILD.bazel

```diff
@@ -18,6 +18,7 @@ ts_project(
     srcs = ["foo.ts"],
     declaration = True,
     tsconfig = "tsconfig.json",
+    validate = False,
```

Comment:
could we change the deps here to `["//examples/dts_lib"]` instead?

Reply:
err no, I guess the point of this example is to use path mapping rather than dep on a linked npm_package rule, so it should be `["//examples/dts_lib:lib"]` - but still doesn't seem like the js_lib should actually be needed in the deps so the validator is telling us a correct thing here?

Reply:
The `js_lib` is a compile-time dependency so no. This is the case that I think will bother people...

Reply:
okay, tried it locally and I see now, `tsc` wants to do the resolution even though the type-checker gives up and makes it an `any`. I think validate = False is correct then 👍🏻